web/js/layers/wv.layers.modal.js

```diff
@@ -367,7 +367,7 @@ wv.layers.modal = wv.layers.modal || function(models, ui, config) {
         redoScrollbar();
         $selectedCategory.iCheck({checkboxClass: 'icheckbox_square-red'});
         $breadcrumb.show();
-        
+
     };
 
     var drawAllLayers = function() {
```

Comment:
@ZachTRice I would work on breaking this method up into smaller functions and add some documentation to the new methods/functions in this component. Ideally we could generate docs using http://usejsdoc.org in the future